src/xdocs/property_types.xml

```diff
@@ -59,7 +59,7 @@
       <p>Anything else will map to <code>false</code>.</p>
     </section>
 
-    <section name="stringSet">
+    <section name="stringCollection">
```

Comment:
unfortunately this update should trigger even more changes:
```
✔ ~/java/github/romani/checkstyle [master|✔] 
$ ag -Q "property_types.html#stringSet" | grep "String Set" | wc -l
43
```

Reply:
I am fine with `stringCollection`.

Reply:
I have written a filter list of classes for this change to be applied, only the checks we are changing in the current PR will be considered. Do you want me to create a separate PR to change all occurences and get that merged before?